README.md

```diff
@@ -1 +1,194 @@
-# blog-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Blog App with Rails</b></h3>
+</div>
+
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [🚀 Video Link](#video-demo)
```

Comment:
- [ ] [OPTIONAL] I can't find any section for video in your ReadMe and this hyperlink is leading nowhere - more like a broken link. Kindly fix this by removing this line to avoid confusion when users try to check the link :wink: